### ISSUE TITLE
Vertical Centering Not Working

### ISSUE THREAD

**Issue by glow12121** [braver/typewriter#31]
Hello,
The vertical centering feature isn't working in Atom. Like, when the screen is centered around the line where you're writing.

I am using Atom in version 1.33.1 x64, on Linux.
Have tried disabling and enabling the package, deleting ~/.atom completely - but to no success.

Could someone help?

**Comment by WispOfTheWinds** [comment]
I don't believe Typewriter is meant to center text vertically in the page, only horizontally, as seen in the screenshot from the readme. ![screenshot](https://camo.githubusercontent.com/681956016f370b2347f18d2505a7615dd0ef114e/687474703a2f2f692e696d6775722e636f6d2f416a6d654643682e706e67)

If it isn't centering _horizontally_ then this should be looked into further.

Side note: I'm not sure if this is actually being maintained anymore, so I forked it recently to mess with. Having an option for vertical centering seems interesting. I might implement that at some point.

**Comment by glow12121** [comment]
> Having an option for vertical centering seems interesting. I might implement that at some point.

I was surprised to find out this feature isn't included in the package: to me, it's the only thing I would expect from such an add-on. Horizontal centering and everything else are secondary.

Vertical centering is included in Zen, but I don't like to use it because I don't need all other features, and not all of them can be disabled.

I guess if someone started a package with the above-mentioned specifications, I could help as a test pilot or something.
